qa-tests-backend/src/test/groovy/NetworkFlowTest.groovy

```diff
@@ -510,8 +510,8 @@ class NetworkFlowTest extends BaseSpecification {
         then:
         "The edge should still be there and it should still be the only edge from A"
         assert edges
-        // Enabling external IPs should not change the number of edges
-        assert edges.size() == 1
+        // Enabling external IPs should include flows for each IP
+        assert edges.size() == 2
```

Comment:
The number of edges will vary from time to time. The reason is that when wget is called on google.com it will be a different IP address each time. The size here will vary depending upon how long the test has run. May the assert should be `edges.size() >= 1`, or the test should be disabled until the API is improved.

Reply:
Done

Reply:
Your change is acceptable, but consider setting the environment variable for the e2e tests such that this test does not need to be changed.